middlewares/validators/user.js

```diff
@@ -14,7 +15,8 @@ const updateUser = async (req, res, next) => {
     linkedin_id: joi.string().optional(),
     twitter_id: joi.string().optional(),
     instagram_id: joi.string().optional(),
-    website: joi.string().optional()
+    website: joi.string().optional(),
+    status: joi.any().valid(...userStatusEnum)
```

Comment:
Do we not provide `.optional()` with this one?

Reply:
Added .optional() for status property as you suggested!

https://github.com/Real-Dev-Squad/website-backend/pull/348/commits/1f0c6c2a1102c0558dfc2d1cf8cfcc4d9a8ab8d5